packages/cryptoplease/lib/l10n/app_en.arb

```diff
@@ -117,14 +117,16 @@
   "back": "Back",
   "unsupportedNft": "Unsupported Nft",
   "yourNfts": "YOUR COLLECTIBLES",
-  "splitKeySecondLinkTitle": "Money is waiting for you",
+  "splitKeySecondLinkTitle": "Money is waiting for you!",
   "splitKeySecondLinkReadyTitle": "Receiving money",
-  "splitKeySecondLinkMessage1": "Please ",
-  "splitKeySecondLinkMessage2": "click now on the second link",
-  "splitKeySecondLinkMessage3": " in the message you received.",
-  "splitKeyTransactionError": "Failed to process the payment",
-  "splitKeyTransactionErrorConsumed": "The money was already withdrawn",
-  "splitKeySuccessMessage": "Payment completed",
+  "splitKeySecondLinkMessage": "Please click the second link in the message you've received.",
+  "splitKeyTransferTitle" : "TRANSFER PROGRESS",
+  "splitKeyTransactionLoading" : "Hang tight. We're almost done...",
+  "splitKeySuccessMessage1" : "SUCCESS!",
+  "splitKeySuccessMessage2" : "Your transfer has been sent.",
+  "splitKeyErrorMessage1": "OOPS!",
+  "splitKeyErrorMessage2": "There was an issue with your transfer.",
+  "splitKeyErrorRetry": "\nPlease retry.",
```

Comment:
There shouldn't be `\n` at the beginning or end of the text string, new line, if needed, should be the responsibility of the code using this string.